src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +15,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
```

Comment:
Let's make this var field, by moving it to the class level. So you will not need to create a new UserService object each time this method is called.